src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,28 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
Getter, Setter, AllArgsConstructor = @Data

Reply:
Dzięki za spostrzeżenie, choć AllArgsConstructor nie zawiera się w @Data, tylko @ReguiredArgsConstructor, także to AllArgsCnstructor muszę zostawić, ale @Getter i @Setter jak najbardziej i już to poprawiam

Reply:
masz rację !:) mój błąd :) nie ma tam AllArgs